tests/python/gaia-ui-tests/gaiatest/tests/functional/settings/test_settings_gps.py

```diff
@@ -25,16 +18,9 @@ def test_enable_gps_via_settings_app(self):
         settings.launch()
 
         # should be on by default
-        self.wait_for_condition(lambda m: settings.is_gps_enabled)
+        settings.wait_for_gps_to_be_enabled()
```

Comment:
This line is not needed. `settings.disable_gps()` should wait until GPS is enabled.